### ISSUE TITLE
[Enhancement] Add support for Header Actions

### ISSUE THREAD

**Issue by jakobe** [kirbydesign/designsystem#2930]
- [X] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
As part of integrating the new `Header` component with `Page` we need to support actions within the header:
![image](https://user-images.githubusercontent.com/9556693/223451057-964bfb1b-580d-4c73-8a0f-7ce483e33b14.png)

## Describe the solution you'd like
Actions can be supplied through a new `<kirby-header-actions>` component with the actions being slotted as `<button kirby-button>` (including support for icons, attentionLevel etc).

These actions should also be shown in the toolbar when scrolling the page as per [the design spec](https://app.zeplin.io/project/60337f8fa4b49814fda1b48a/screen/6399d4f89a3811140bcc90d5)
![image](https://user-images.githubusercontent.com/9556693/223451248-92db80b7-018b-4473-a405-dce0753671ef.png)

To support this we need the consuming code to tag the actions with a `*kirbyPageActions` directive, as this is [the only way to render the content in 2 places](https://github.com/angular/angular/issues/9173#issuecomment-225732419) 

## Have you considered any alternatives?
No.

## Are there any additional context?
Note: The collapsing of buttons into a "more" menu button as shown in the design specs will be handled by a separate issue/PR.

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [x] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [x] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [x] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [x] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".
